pkg/util/util_test.go

```diff
@@ -43,6 +43,16 @@ var prettyDiffTest = []struct {
 	{[]int{1, 2, 3, 4}, []int{1, 2, 3, 4}, ""},
 }
 
+var isEqualIgnoreOrderTest = []struct {
+	inA      []string
+	inB      []string
+	outEqual bool
+}{
+	{[]string{"a", "b", "c"}, []string{"a", "b", "c"}, true},
+	{[]string{"a", "b", "c"}, []string{"a", "c", "b"}, true},
+	{[]string{"a", "b"}, []string{"a", "c", "b"}, false},
```

Comment:
another test case would be "equal length / unequal content"
```go
[]string{"a", "b", "c"}, []string{"a", "d", "c"}, true
```
The second slice has `d` instead of `b`